sympy/concrete/tests/test_gosper.py

```diff
@@ -133,21 +134,21 @@ def test_gosper_sum_AeqB_part1():
 
 def test_gosper_sum_AeqB_part2():
     f2a = n**2*a**n
-    f2b = (n - r/2)*binomial(r, n)
+    f2b = (n - k/2)*binomial(k, n)
     f2c = factorial(n - 1)**2/(factorial(n - x)*factorial(n + x))
 
     g2a = -a*(a + 1)/(a - 1)**3 + a**(
-        m + 1)*(a**2*m**2 - 2*a*m**2 + m**2 - 2*a*m + 2*m + a + 1)/(a - 1)**3
-    g2b = (m - r)*binomial(r, m)/2
+        n + 1)*(a**2*n**2 - 2*a*n**2 + n**2 - 2*a*n + 2*n + a + 1)/(a - 1)**3
+    g2b = (n - k)*binomial(k, n)/2
     ff = factorial(1 - x)*factorial(1 + x)
     g2c = 1/ff*(
-        1 - 1/x**2) + factorial(m)**2/(x**2*factorial(m - x)*factorial(m + x))
+        1 - 1/x**2) + factorial(n)**2/(x**2*factorial(n - x)*factorial(n + x))
 
-    g = gosper_sum(f2a, (n, 0, m))
+    g = gosper_sum(f2a, (n, 0, n))
     assert g is not None and simplify(g - g2a) == 0
-    g = gosper_sum(f2b, (n, 0, m))
+    g = gosper_sum(f2b, (n, 0, n))
     assert g is not None and simplify(g - g2b) == 0
-    g = gosper_sum(f2c, (n, 1, m))
+    g = gosper_sum(f2c, (n, 1, n))
```

Comment:
Can we avoid the variable replacement in this file?

Reply:
The variables were replaced because they needed the assumptions.